redfish-core/lib/pcie_slots.hpp

```diff
@@ -611,10 +481,81 @@ inline void getValidPCIeSlotList(
             return slot1.first < slot2.first;
         });
 
-        callback(slotPathConnNames);
+        callback(ec, slotPathConnNames);
     });
 }
 
+inline void afterHandlePCIeSlotCollectionGet(
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+    const std::string& chassisID,
+    const std::optional<std::string>& validChassisPath)
+{
+    if (!validChassisPath)
+    {
+        BMCWEB_LOG_WARNING("Not a valid chassis ID:{}", chassisID);
+        messages::resourceNotFound(asyncResp->res, "Chassis", chassisID);
+        return;
+    }
+
+    BMCWEB_LOG_DEBUG("Get properties for PCIeSlots associated to chassis = {}",
+                     chassisID);
+
+    asyncResp->res.jsonValue["@odata.type"] = "#PCIeSlots.v1_5_0.PCIeSlots";
+    asyncResp->res.jsonValue["Name"] = "PCIe Slot Information";
+    asyncResp->res.jsonValue["@odata.id"] =
+        boost::urls::format("/redfish/v1/Chassis/{}/PCIeSlots", chassisID);
+    asyncResp->res.jsonValue["Id"] = "1";
```

Comment:
why is Id 1 instead of PCIeSlots?

Reply:
Do collections normally have IDs?

Reply:
Oh well, it was there before

Reply:
Upstream and 1060 all have the same Id = 1.
What other value may be better as Id?  Or, would it better to remove it?

Reply:
Would it be better to change to "PCIeSlots"

Reply:
>Gunnar help to show that "Id" is not used for the other collections and also dmtf mockup does not use it.
>https://github.com/openbmc/bmcweb/blob/master/redfish-core/lib/processor.hpp#L1291
>Yeah, there mockups don't have it https://redfish.dmtf.org/redfish/mockups/v1/1448

So, I'll remove it